src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html

```diff
@@ -103,3 +104,14 @@
 {% if swiss_qrbill %}
 <link rel="stylesheet" href="{% static "pretixplugins/banktransfer/swisscross.css" %}">
 {% endif %}
+
+<hr>
+<form class="form-inline helper-display-inline" method="post"
+        action="{% eventurl event "plugins:banktransfer:mail_invoice" order=order.code secret=order.secret %}">
+    {% csrf_token %}
+    <input type="email" name="email" class="form-control" value=""/>
+    <button class="btn btn-default">
+        {% trans "Send invoice via E-Mail" %}
```

Comment:
We're not as consistent with this as we should be, but we tend to use:

```suggestion
        {% trans "Send invoice via email" %}
```